lib/services/appmanager-service.ts

```diff
@@ -76,7 +76,7 @@ class AppManagerService implements IAppManagerService {
 				this.$logger.warn("You have not set the --publish switch. Your users will not receive a push notification.");
 			}
 
-			let uploadedAppData: Server.UploadedAppData = this.$server.tam.uploadApplication1(projectName, projectName, projectPath, publishSettings).wait();
+			let uploadedAppData: Server.UploadedAppData = this.$server.tam.uploadApplicationFromUri(projectName, projectName, <any>projectPath, publishSettings).wait();
```

Comment:
Since this `<any>projectPath` is working we need to check why the `uploadApplicationFromUri` is generated with `packageUri: Server.Uri` instead of `packageUri: string`